filter sensitive headers & cookies

Kelvin4664 opened the pull request:
- Added 'CSRF_COOKIE' to default filter_params
- Added filtering to meta attributes for flask or & django
- Added filtered cookies to cgi_data for flask & django
- removed conversion of request.META to dict because it is a dict by default

Fixes #43 
Fixes #44

joshuap commented on honeybadger/contrib/django.py:
```diff
@@ -74,13 +74,16 @@ def generate_payload(self, config, context):
             payload['session'] = filter_dict(dict(request.session), config.params_filters)
 
+        if hasattr(request, 'COOKIES'):
+            payload['cgi_data']['COOKIES'] = filter_dict(request.COOKIES, config.params_filters)
```
This is the key that we should use in the JSON payload:

```suggestion
            payload['cgi_data']['HTTP_COOKIE'] = filter_dict(request.COOKIES, config.params_filters)
```

joshuap commented on honeybadger/contrib/flask.py:
```diff
@@ -51,15 +51,16 @@ def generate_payload(self, config, context):
         cgi_data.update({
-            'REQUEST_METHOD': _request.method
+            'REQUEST_METHOD': _request.method,
+            'COOKIES': dict(_request.cookies)
```
```suggestion
            'HTTP_COOKIE': dict(_request.cookies)
```

joshuap reviewed (commented):
Looking good @Kelvin4664. 🎉

Just a few suggestions:

1. See suggested changes to cgi_data key names
2. Can you add a [changelog](https://github.com/honeybadger-io/honeybadger-python/blob/master/CHANGELOG.md) entry under the "Unreleased" heading? [Here's an example entry that I added recently](https://github.com/honeybadger-io/honeybadger-python/commit/3058f047cb56ceb7b3d51703f8506bd240c61075#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed).
3. In the future it would help to reference the issues that this is fixing in the PR description. (I went ahead and updated the description on this one)

Thanks!

Kelvin4664 commented:
> Looking good @Kelvin4664. 🎉
> 
> Just a few suggestions:
> 
> 1. See suggested changes to cgi_data key names
> 2. Can you add a [changelog](https://github.com/honeybadger-io/honeybadger-python/blob/master/CHANGELOG.md) entry under the "Unreleased" heading? [Here's an example entry that I added recently](https://github.com/honeybadger-io/honeybadger-python/commit/3058f047cb56ceb7b3d51703f8506bd240c61075#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed).
> 3. In the future it would help to reference the issues that this is fixing in the PR description. (I went ahead and updated the description on this one)
> 
> Thanks!

Thanks Josh..
All resolved.

joshuap reviewed (approved):
Thanks!
